### PR TITLE
keep customize folder

### PR DESCRIPTION
**Pull request by rodinux**
## Problem

- *customize folder is sometimes edited, the upgrade could delete the content*

## Solution

- *keep the folder /customize*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by rodinux** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -90,7 +90,7 @@ then
 	ynh_script_progression --message="Upgrading source files..." --weight=1
 
 	# Download, check integrity, uncompress and patch the source from app.src
-	ynh_setup_source --dest_dir="$final_path" --keep="config/config.js"
+	ynh_setup_source --dest_dir="$final_path" --keep="config/config.js /customize"
```
Comment (ericgaspar) [inline scripts/upgrade:93]: I don't know if we need `/`

Comment (rodinux) [reply scripts/upgrade:93]: I am talking about a folder needed to customize the application, in my instance I have edited few files in this directory... https://docs.cryptpad.org/fr/admin_guide/customization.html#look-and-feel

**Comment by rodinux** [comment]
You are right, is better like this `ynh_setup_source --dest_dir="$final_path" --keep="config/config.js customize/"` ?

**Pull request merged**